### ISSUE TITLE
Computer freeze when open folder

### ISSUE THREAD

**Issue by condor2** [notepadqq/notepadqq#447]
I have tried to open a folder with 10-13 files and computer freeze.

I5-4440 with 8GB RAM, Arch Linux 64bit

Notepadqq 1.0.1
Qt version 5.8.0

**Comment by JuBan1** [comment]
How did you try to open a folder? Using `File->Open Folder`?

At least for me there's an issue with this option: it doesn't try to open the selected folder but the open folder. So let's say you click `Open Folder` and navigate to `/user/My Files/`, there you select the `Scripts` folder and click `Ok`, Nqq is supposed to open the `/user/My Files/Scripts` folder but will actually open the `/user/My Files` folder instead. And if you have lots of files in this folder Nqq will open them all; that can freeze your computer.

Can you verify if that is what happens?

**Comment by condor2** [comment]
Yes I tried using File-->Open Folder and was 19 files in a folder test and computer freeze.

Notepad++ (in wine) open that folder instantly
